README.md

```diff
@@ -28,11 +28,8 @@ To list all the available tasks for running:
 
 ## Prerequisites
 
-1. A [msgbusd](https://github.com/threefoldtech/rmb_go) instance must be running on the node. This client uses RMB (message bus) to send messages to nodes, and get the responses.
-2. A valid MNEMONICS.
-3. [yggdrasil](https://yggdrasil-network.github.io/installation.html) service running with a valid ip assigned to the MNEMONICS on [polkadot](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ftfchain.dev.grid.tf%2Fws#/accounts).
-4. Golang compiler > 1.13 to run the grid proxy server.
-5. Postgres database
+1. Golang compiler > 1.13 to run the grid proxy server.
```

Comment:
The compiler shouldn't be a requirement, always link to the release binaries on github, you probably need a separate developer doc e.g wiki/devsetup.md